components/form-builder/layout/Publish.tsx

```diff
@@ -34,6 +37,7 @@ export const Publish = () => {
     if (result && result?.error) {
       setError(true);
       return;
+      s;
```

Comment:
typo?

Reply:
Just pushed a commit to fix.  A stray keyboard command gone awry.